### PR TITLE
Dispatch event to switch on ?tab=about tab when resultCount is 0

### PR DESCRIPTION
**Pull request by nsharma123**
dispatch an event to collection-page if resultcount is zero. The event will be listened at offshoot and switch to ?tab=about to show about page of collection.

### CODE REVIEW

File: src/collection-browser.ts [hunk 1]
```diff
@@ -1125,6 +1125,14 @@ export class CollectionBrowser
...
   }
 
+  private emitTabChanged(tabId: string) {
+    this.dispatchEvent(
+      new CustomEvent<String>('tabChanged', {
+        detail: tabId,
+      })
+    );
+  }
+
```
Comment (latonv) [inline src/collection-browser.ts:1135]: Suggestion: Let's make it so collection browser doesn't need to have knowledge of its parent component's tabs. Maybe we just emit an `emptyResults` event with no args, and let the parent handle that as appropriate?

Comment (nsharma123) [reply src/collection-browser.ts:1135]: I used custom event with args to make is reusable if future if we want to communicate with offfshoot later to switch between tabs. that way, we can simply send tabId to collection-browser to switch.

Comment (iisa) [reply src/collection-browser.ts:1135]: +1 to emptyResults bc tab is not direct child of collection browser, it's slotted 👍 please do update to emptyResults

**Review by iisa — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "description": "The Internet Archive Collection Browser.",
   "license": "AGPL-3.0-only",
   "author": "Internet Archive",
-  "version": "1.13.0",
+  "version": "1.13.0-alpha3",
```
Comment (latonv) [inline package.json:6]: Just a reminder to revert this version change before merging.

**Review by latonv — APPROVED** [review]
Thanks for updating the event, LGTM!

**Review by nsharma123 — COMMENTED** [review]
Thanks. Changed.

**Pull request merged**